extension/options/optionsView.jsx

```diff
@@ -3,29 +3,51 @@
 
 import * as browserUtil from "../browserUtil.js";
 
+export const TABS = {
+  GENERAL: 0,
+  NICKNAMES: 1,
+};
+
 export const Options = ({
   inDevelopment,
   version,
   keyboardShortcutError,
   userOptions,
   userSettings,
   updateUserSettings,
+  updateTabValue,
+  tabValue,
+  updateNickname,
+  registeredNicknames,
 }) => {
   return (
     <div className="settings-page">
-      <LeftSidebar version={version} />
-      <RightContent
+      <LeftSidebar version={version} updateTabValue={updateTabValue} />
+      <General
         inDevelopment={inDevelopment}
         keyboardShortcutError={keyboardShortcutError}
         userOptions={userOptions}
         userSettings={userSettings}
         updateUserSettings={updateUserSettings}
-      />
+        active={tabValue === TABS.GENERAL}
```

Comment:
Instead of this I'd rather this function do:

```js
{tabValue === TABS.GENERAL ?
  <General ... />
  : null}
```

That is, it shouldn't be the component's responsibility to null itself out, instead have this include only the right components.